### DISCUSSION TITLE
2/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#96] [Daily Sprint]
Buen día
¿Qué hice ayer?
¿Qué estoy haciendo?
¿Impedimento?

**Comment by Pamela198713** [comment]
Ayer nada. Feriado.
Hoy hay clases y veré qué tareas hay de web.
Tiempo y organización en la casa.

**Comment by rpgrca** [comment]
Ayer armé el proyecto de la app y empecé a armar algunas cosas con los modelos que se van a usar. Hoy durante o después de la clase espero continuar con el resto de los modelos con sus pruebas y ver si puedo empezar con los servicios para conectarse al servidor web (que por ahora van a retornar cosas hardcodeadas pero ya dejarlo preparado para cuando haya que correr los dos al mismo tiempo).

**Comment by LuciaL152** [comment]
Ayer hice muy poco estuve viendo como era el proyecto antes y revisando el aula.
Hoy estuve armando unos wireframes para ver como podría verse la app y ahora voy a estar en la clase.
Tiempo.

**Comment by fmdcba** [comment]
Ayer me tomé todo el día del trabajador con la familia.
Hoy estuve presente en la clase y bajé los cambios que introdujo Roberto para correr la app mobile. Intentaré agregar algunas pantallas y continuaré analizando el repo enbalde y su doc.
